pkg/daemons/config/types.go

```diff
@@ -140,6 +141,7 @@ type CriticalControlArgs struct {
 	DisableNPC            bool         `cli:"disable-network-policy"`
 	DisableServiceLB      bool         `cli:"disable-service-lb"`
 	EncryptSecrets        bool         `cli:"secrets-encryption"`
+	MultiClusterCIDR      bool         `cli:"multi-cluster-CIDR"`
```

Comment:
Nit: CLI flags are all lowercase
```suggestion
	MultiClusterCIDR      bool         `cli:"multi-cluster-cidr"`
```